### PR TITLE
P4RT-2.1: delete table entry after successful write

### PR DESCRIPTION
**Pull request by kjahed**
Table entry must be deleted after successful write to avoid subsequent subtests from failing.

### CODE REVIEW

File: feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go [hunk 1]
```diff
@@ -276,6 +276,10 @@ func canWrite(t *testing.T, args *testArgs) (bool, error) {
 			t.Logf("Expected P4Client error: Client write permission (highID %d, lowID %d): want %v, got %v", args.highID, args.lowID, args.wantWrite, false)
 		}
 		return false, writeErr
+	} else {
```
[the remark this answers is not in the document]
Comment (kjahed) [reply feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go]: Right..fixed.

**Review by Ankur19 — DISMISSED** [review]

**Comment by kjahed** [comment]
Added delay after removeClient to allow some time for the client to disconnect. Otherwise, in some tests (e.g., TestPrimaryReconnect), the backup client could receive the incorrect status code (ALREADY_EXISTS) rather than (NOT_FOUND).

**Review by rszarecki — DISMISSED** [review]

**Review by prinikasn — COMMENTED** [review]
@kjahed  please resolve the merge conflict

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by prinikasn — DISMISSED** [review]

**Pull request merged**